sympy/core/expr.py

```diff
@@ -3492,7 +3494,7 @@ def leadterm(self, x, logx=None, cdir=0):
         c = c.subs(d, log(x))
         return c, e
 
-    def as_coeff_Mul(self, rational=False):
+    def as_coeff_Mul(self, rational: bool = False) -> tTuple['Number', 'Expr']:
```

Comment:
Not your fault, but the consequences on keyword arguments are, well, not my favorite part of type checking...